ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -208,6 +210,42 @@ public void handlePost(RestRequest restRequest, RestResponseChannel restResponse
   }
 
   @Override
+  public void handlePut(RestRequest restRequest, RestResponseChannel restResponseChannel) {
```

Comment:
ambry does not support PUTs. You should do the same thing you did in `AdminBlobStorageService` here.